### ISSUE TITLE
Vitest >=0.34.3 throws error cannot find package '__sveltekit'

### ISSUE THREAD

**Issue by olafurw** [vitest-dev/vitest#4067]
### Describe the bug

Same/similar error message as #3483 but it works in 0.34.2

### Reproduction

Happens on my work project and I was able to make it happen by cloning the repro from #3483 and updating the package version. If you move it down to 0.34.2 and run `npm install && npm run test:run` it works but not with 0.34.3

https://stackblitz.com/edit/vitest-dev-vitest-v9c8dj?file=package.json

### System Info

```shell
System:
    OS: Linux 6.4 Fedora Linux 38 (Workstation Edition)
    CPU: (16) x64 12th Gen Intel(R) Core(TM) i7-1260P
    Memory: 21.10 GB / 30.95 GB
    Container: Yes
    Shell: 5.2.15 - /bin/bash
  Binaries:
    Node: 18.16.0 - ~/.nvm/versions/node/v18.16.0/bin/node
    npm: 9.7.2 - ~/.nvm/versions/node/v18.16.0/bin/npm
  Browsers:
    Chrome: 116.0.5845.140
  npmPackages:
    @vitest/coverage-v8: ^0.34.2 => 0.34.2 
    vite: ^4.4.9 => 4.4.9 
    vitest: ^0.34.3 => 0.34.3
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by sheremet-va** [comment]
Looks like it's fixed in 0.34.4. Can you confirm?

**Comment by olafurw** [comment]
Yes update fixed it. Sorry for the delayed reply.
